python/samples/demos/guided_conversations/interactive_guided_conversation.py

```diff
@@ -80,12 +79,12 @@ async def main() -> None:
 
     kernel = Kernel()
     service_id = "gc_main"
-    token_provider = get_bearer_token_provider(DefaultAzureCredential(), "https://cognitiveservices.azure.com/.default")
+    token_endpoint = "https://cognitiveservices.azure.com/.default"
```

Comment:
Small note: this isn't needed -- it's set by default on the Azure OpenAI settings: 

https://github.com/microsoft/semantic-kernel/blob/52ceba5da17390e0839148c59e2fbbb407213dc8/python/semantic_kernel/connectors/ai/open_ai/settings/azure_open_ai_settings.py#L78

Reply:
Oh awesome, done!